data/hackathon/scenario5.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Single Hybrid Cloud Platform for AI and Intelligent Apps
+exercise: 3
```

Comment:
Can you please update this to `exercise: 5`.

Reply:
sure